app/uk/gov/hmrc/gform/auditing/AuditService.scala

```diff
@@ -112,4 +111,23 @@ trait AuditService {
     )
   }
 
+  def sendSubmissionEventHashed(hashedValue: String)(implicit hc: HeaderCarrier, ec: ExecutionContext) = {
```

Comment:
The submission event needs to include the hashedvalue string as well as the hash